wiki/Community/osu!monthly/en.md

```diff
@@ -0,0 +1,53 @@
+# osu!monthly
+
+**osu!monthly** is a monthly newsletter managed by the [osu!monthly Team](/wiki/People/osu!monthly_Team), with the goal of summarising the various community and official events, news, announcements, etc. that occur every month. The project has been led by several other groups of community members in the past, with the goal always being the same: inform the average player about the previous and current events in order to shed light over the entire osu! community.
```

Comment:
```suggestion
**osu!monthly** is a monthly newsletter managed by the [osu!monthly team](/wiki/People/osu!monthly_Team), with the goal of summarising the various community and official events, news, announcements, etc. that occur every month. The project has been led by several other groups of community members in the past, with the goal always being the same: inform the average player about the previous and current events in order to shed light over the entire osu! community.
```

Reply:
if you accepted my other suggestion about spelling `osu!monthly team` with all lowercase, you also need to do it here for consistency

also, @Walavouchey I think we should rename the `People/osu!monthly_Team` directory to lowercase too, since there isn't a team for which this is an exact name, it's just a noun: https://osu.ppy.sh/wiki/en/People/osu%21_wiki_maintainers

Reply:
https://discordapp.com/channels/188630481301012481/1003689172420276246/1062416922664243260 convo about capital vs non-capital T, this was already discussed but feel free to weigh in yourself in the chat

Reply:
it's a weird one since other groups with "team" in them are capitalised, except "osu! team". i don't have much of an opinion here though.